docs/atlascli/command/atlas-logs-download.txt

```diff
@@ -76,7 +76,7 @@ Options
    * - --out
      - string
      - false
-     - Optional output file name. Uses the log name if the output file's name is not specified.
```

Comment:
it's optional for this comamnd, this was the problem, the reuse of the string constant